root.tf

```diff
@@ -247,27 +247,30 @@ module "backend_code_s3" {
 }
 
 module "ecr_yara_repository" {
-  source      = "./tdr-terraform-modules/ecr"
-  name        = "yara"
-  common_tags = local.common_tags
-
+  source           = "./tdr-terraform-modules/ecr"
+  name             = "yara"
+  image_source_url = "https://github.com/nationalarchives/tdr-antivirus/blob/master/Dockerfile-yara"
+  common_tags      = local.common_tags
 }
 
 module "ecr_dependencies_repository" {
-  source      = "./tdr-terraform-modules/ecr"
-  name        = "yara-dependencies"
-  common_tags = local.common_tags
+  source           = "./tdr-terraform-modules/ecr"
+  name             = "yara-dependencies"
+  image_source_url = "https://github.com/nationalarchives/tdr-antivirus/blob/master/Dockerfile-dependencies"
+  common_tags      = local.common_tags
 }
 
 module "ecr_rules_repository" {
-  source      = "./tdr-terraform-modules/ecr"
-  name        = "yara-rules"
-  common_tags = local.common_tags
+  source           = "./tdr-terraform-modules/ecr"
+  name             = "yara-rules"
+  image_source_url = "https://github.com/nationalarchives/tdr-antivirus/blob/master/Dockerfile-compile"
```

Comment:
@MancunianSam Have I linked to the right Dockerfiles for the three yara repos?

Reply:
Yep, they're right.

Reply:
Ta. Could you take a look at the rest of the PR to see if it's OK to merge?